src/component/util.js

```diff
@@ -0,0 +1,20 @@
+// Copyright 2015-present Drifty Co.
+// http://drifty.com/
```

Comment:
I wasn't sure about that. I thought since both projects are MIT licensed have the copyright there would be sufficient. But I'm not a lawyer 😛

I'm sure it's fine.